tests/integrations/raftstore/test_conf_change.rs

```diff
@@ -79,8 +79,8 @@ fn test_simple_conf_change<T: Simulator>(cluster: &mut Cluster<T>) {
     // peer 3 must exist
     must_get_equal(&engine_3, b"k3", b"v3");
 
-    // add peer 2 then remove it again.
-    pd_client.must_add_peer(r1, new_peer(2, 2));
+    // add peer 22 then remove it again.
```

Comment:
Why change this?

Reply:
Please ignore, this is cherry-picked from the previous changes, I have deleted it.